src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java

```diff
@@ -0,0 +1,20 @@
+/*
+WhitespaceAfter
+tokens = LITERAL_TRY
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputWhitespaceAfterLiteralTry {
+    public static void main(String[] args) throws IOException {
+        try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
```

Comment:
Put one more line with space after try

Reply:
Changes Done

Reply:
Keep example of separation by new line.
Iny comment I mentioned to add one more line to extend test cases